packages/react-components/src/atoms/TabLink.tsx

```diff
@@ -27,11 +28,37 @@ const activeStyles = ({ colors: { primary500 = fern } = {} }: Theme) =>
     fontWeight: 'bold',
   });
 
+const iconStyles = css({
+  display: 'inline-grid',
+  verticalAlign: 'middle',
+  paddingRight: `${6 / perRem}em`,
+});
+
 interface TabLinkProps {
   readonly href: string;
+  readonly Icon?: ComponentType<IconProps>;
   readonly children: ReactNode;
 }
-const TabLink: React.FC<TabLinkProps> = ({ href, children }) => {
+const TabLink: React.FC<TabLinkProps> = ({ href, children, Icon }) => {
+  const active =
+    new URL(href, window.location.href).pathname === window.location.pathname;
```

Comment:
When user metric is selected the tab appears as active

![Screenshot 2024-04-01 at 13 01 47](https://github.com/yldio/asap-hub/assets/16595804/7af673fc-32a9-40e9-a79e-4777d6d91ae2)


But when team is selected, it doesn't appear as active
![Screenshot 2024-04-01 at 13 01 53](https://github.com/yldio/asap-hub/assets/16595804/0086149b-0521-436d-bb73-d69b5f2fb928)


Maybe there's something wrong in the way it's being called?